"cli" dep package API broke the build

Found when fixing #11 

```shell
vbatts@melisma:/tmp/tmp.JH8RIWziTw$ go get github.com/vbatts/sl-feeds/cmd/sl-feeds
# github.com/vbatts/sl-feeds/cmd/sl-feeds
src/github.com/vbatts/sl-feeds/cmd/sl-feeds/main.go:24:17: cannot use cli.StringFlag literal (type cli.StringFlag) as type cli.Flag in slice literal:
	cli.StringFlag does not implement cli.Flag (Apply method has pointer receiver)
src/github.com/vbatts/sl-feeds/cmd/sl-feeds/main.go:28:17: cannot use cli.StringFlag literal (type cli.StringFlag) as type cli.Flag in slice literal:
	cli.StringFlag does not implement cli.Flag (Apply method has pointer receiver)
src/github.com/vbatts/sl-feeds/cmd/sl-feeds/main.go:32:15: cannot use cli.BoolFlag literal (type cli.BoolFlag) as type cli.Flag in slice literal:
	cli.BoolFlag does not implement cli.Flag (Apply method has pointer receiver)
src/github.com/vbatts/sl-feeds/cmd/sl-feeds/main.go:36:15: cannot use cli.BoolFlag literal (type cli.BoolFlag) as type cli.Flag in slice literal:
	cli.BoolFlag does not implement cli.Flag (Apply method has pointer receiver)
```

--- a/go.mod
+++ b/go.mod
@@ -1,0 +1,10 @@
+module github.com/vbatts/sl-feeds
+
+go 1.13
+
+require (
+	github.com/BurntSushi/toml v0.3.1
+	github.com/gorilla/feeds v1.1.1
+	github.com/kr/pretty v0.2.1 // indirect
+	github.com/urfave/cli v1.22.5
+)

